core/domain/classifier_services.py

```diff
@@ -305,9 +313,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, exp_id,
```

Comment:
@seanlip . I have implemented the mark_job_complete function, but just made it use the already existing functions, but the controller ends up using only the mark_job_complete function. PTAL and tell me if its fine.